index.html

```diff
@@ -451,10 +451,27 @@ <h2 id="format">The did:orb Format</h2>
           [[MULTIHASH]] and [[MULTIBASE]] defines the multihash and multibase formats.
         </p>
 
+        <p>
+          The canonical Orb DID includes a multihash of the latest anchor object that contains a create or recovery operation for the Sidetree DID suffix.
+          The first segment after `did:orb` contains the anchor multihash and the second segment contains the Sidetree DID suffix.
+        </p>
+
         <pre class="example" title="A canonical Orb DID">
           did:orb:uEiDlXjleTwr4eZalpXVy086zs-TPK-h54ojbpl7EBvZeHQ:EiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg
         </pre>
 
+        <p>
+          In cases where the anchoring information is not included into the DID string, the graph multihash will be represented by the identity code (`0x00`) with a 0 length output.
+          The base64url representation of an unknown anchor object is: `uAAA`.
```

Comment:
note: `baaaa`, `z11` and `uAAA` are equivalent (base32 vs base58btc vs base64url).